client/nginx.go

```diff
@@ -140,6 +141,20 @@ type Connections struct {
 	Idle     uint64
 }
 
+// Slabs is map of slab stats by zone name.
+type Slabs map[string]Slab
+
+// Slab represents slab related stats.
+type Slab struct {
```

Comment:
@faisal-memon 
was it intentional to not include slots stats?

Reply:
good catch. i'm not using slots stats but Ill add that in.

Reply:
done